tokens/crf.json

```diff
@@ -1,6 +1,6 @@
 {
     "ethereum_address": "0x508df5aa4746be37b5b6a69684dfd8bdc322219d",
-    "aurora_address": "",
+    "aurora_address": "0x026dda7f0f0a2e42163c9c80d2a5b6958e35fc49",
```

Comment:
Please make sure to add it in the initial PR. I already added this and the icon in #78